web/compose-compiler-integration/build.gradle.kts

```diff
@@ -0,0 +1,134 @@
+fun cloneTemplate(templateName: String, contentMain: String, contentLib: String): File {
```

Comment:
If I got what's going on here right (and I apologize if not) it seems to me that just using copy task would be a better thing to invest into. Reason is that Copy task, among other things, sets inputs and outputs - that said, the task will be triggered only when relevant changes happened.